dbg.h

```diff
@@ -550,10 +550,15 @@ inline bool pretty_print(std::ostream& stream, const time&) {
   const auto us =
       duration_cast<microseconds>(now.time_since_epoch()).count() % 1000000;
   const auto hms = system_clock::to_time_t(now);
+#if _MSC_VER >= 1600
+  const auto tm = new std::tm;
+  localtime_s(tm, &hms);
+#else
   const std::tm* tm = std::localtime(&hms);
+#endif
   stream << "current time = " << std::put_time(tm, "%H:%M:%S") << '.'
          << std::setw(6) << std::setfill('0') << us;
-
+  delete tm;
```

Comment:
Now you are calling `delete` on a `tm*` pointer to a static `tm` object inside `std::localtime` when the `#else` branch is active. Please don't use `new`/`delete` at all here but rather simply "stack-allocate" a `tm` object locally.